errors/error.go

```diff
@@ -1,4 +1,4 @@
-package multipart
+package errors
```

Comment:
This is pretty breaking change. In the future it's smarter to supply both until one has been phased our correctly. This will break other projects.

Reply:
multipart is new, and only used by the fuga package in lm =)